### PR TITLE
Add recipe for ISA-tools

### PR DESCRIPTION
**Pull request by RJMW**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/isatools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  version: "0.9.3"
+
+source:
+  fn: isatools-0.9.3.tar.gz
+  url: https://pypi.python.org/packages/a0/80/18418311f29a076d89b5b5164264231d7e9b29d3e5e647716cfd34dd708d/isatools-0.9.3.tar.gz
+  md5: 71c3644fcee7743e3cd3062481cb3b40
+
+build:
+  skip: True # [py27]
+  number: 1
```
Comment (bgruening) [inline recipes/isatools/meta.yaml]: please start with 0

File: recipes/isatools/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt 
+
+# Add more build steps here, if they are necessary.
+
```
Comment (bgruening) [inline recipes/isatools/build.sh]: can you remove the comments?

Comment (RJMW) [reply recipes/isatools/build.sh]: Yes, I can.

File: recipes/isatools/meta.yaml (hunk 1 above)
Comment (RJMW) [reply recipes/isatools/meta.yaml]: Done

**Pull request merged**